book/eha.qmd

```diff
@@ -81,17 +96,73 @@ $$\delta_{ie} = \II(Y_i \leq C_i \wedge E_i = e) \in \{0, 1\},\ e=1,\ldots,k$$
+
+
+
+## Recurrent Events {#sec-recurrent-events}
 
-![Illustration of transitions in the recurrent events setting. Left: "Clock reset" representation of recurrent events. Each time an event occurs, the clock is reset to 0 and we once again wait for occurence of an event (of the same type); Right: "Clock forward" representation. A subject transitions from 1-st, to 2-nd, etc. to K-th event occurence](Figures/survival/recurrent-events.png){#fig-recurrent-events fig-alt="Schematic illustration of recurrent events transitions."}
+The *recurrent events* setting is defined by the same event type occurring multiple times.
+While the recurrent events setting can be viewed as a special case of the multi-state setting (@sec-multi-state), we treat it here separately as it can also be approached as single event data with correlation.
+Some examples of recurrent events include recurrence of infectious diseases, episodes of epilepsy, and replacements of machine parts.
+
+@tbl-cgd-data shows data of chronic granolotomous disease (CGD) for two subjects.
+Note the start-stop notation of the data ($t_{start}, t_{stop}$ respectively) that indicates the time in which a subject enters the risk set for the k*th* event number.
+The column $\delta$ indicates whether the subject experienced the $k$th outbreak or was censored.
+For example, Subject 1 (first three rows) experienced the first outbreak (first row) after 219 days, at which time they entered the risk set for event number 2.
+The second event (second row) was experienced at 373 days and the subject was then censored for the third event (third row) after 414 days.
+This notation is equivalent to a multi-state representation of the recurrent events process, where each event number represents a different state.
+In @fig-recurrent-events, right panel, this is refered to as "Clock forward" approach.
+
+| ID| $k$ | $t_{start}$| $t_{stop}$|$\delta$|$gap$|
+|--:|---:|---:|------:|--:|-----:|
+|  1|1|  0 | 219|   1| 219| 
+|  1|2|219 | 373|   1| 154| 
+|  1|3|373 | 414|   0|  41| 
+|  7|1|  0 | 292|   1| 292| 
+|  7|2|292 | 364|   0|  72| 
+: Subset of the `cgd`  [@pkgsurvival] time-to-event dataset.
+Rows are individual observation units ($ID$), $k$ is the $k$th event occurrence, $t_{start}$ and $t_{stop}$ mark the beginning and end of the "at risk time" for the $k$th event occurence, $\delta$ is the event indicator and $gap = t_{stop} - t_{start}$. {#tbl-cgd-data}
+
+For recurrent events data without competing (terminal) event, however, the "Clock reset" approach (@fig-recurrent-events, left panel) is also popular. Instead of representing the data in the start-stop notation, the gap time is calculated instead (see fourth column in @tbl-cgd-data).
+In this representation, we only consider how long the subject was at risk for the $k$th event, not at which time (on the calendar time scale).
+It has the advantage that standard methods for analysis of single-event data can be used for modeling.
```

Comment:
I don't understand why this is the case, I think more info or an example is needed to explain this

Reply:
Like how can I use standard models if they're correlated? Stratified sampling, etc?